sheet_to_triples/trans.py

```diff
@@ -67,21 +69,23 @@ class Transform:
         'book',
         'sheet',
         'lets',
+        'conds',
```

Comment:
Not sure this needs to be a different top level var from `lets`?

Reply:
It seems nicer to me than having type-checking for whether it's a string or tuple and invoking `lets` behaviour or `conds` behaviour based on that. I think having two separate variables where the things you pass it are one shape, and one shape only, is clearer.